buildpipeline/windows.groovy

```diff
@@ -74,7 +74,7 @@ simpleNode('Windows_NT','latest') {
                 {
                     targetHelixQueues = ['Windows.7.Amd64.Open',
                                          'Windows.81.Amd64.Open',
-                                         'Windows.10.Amd64.ClientRS3.ES.Open',]
+                                         'Windows.10.Amd64.ClientRS4.ES.Open',]
```

Comment:
For RS4 we don't have an ES image. If you need an Spanish image we'll have to stick with RS3 for now and start working in an RS4.ES. If ES is not a requirement please just use 'Windows.10.Amd64.ClientRS4.Open'

Reply:
ES is a requirement. I suggest we move to RS3 and yes it would be great to create an RS4.ES. Shall I open a core-eng issue JC?

Reply:
Yes please. Go ahead an open it and I'll make sure this is taken care of.

Reply:
done thanks